lld/test/ELF/emulation-hexagon.s

```diff
@@ -0,0 +1,39 @@
+# REQUIRES: hexagon
```

Comment:
emulation-systemz.s is the latest `emulation-*` test. We use `llvm-readelf` there for better readability.

Reply:
Ok, updated to use `llvm-readelf`.